src/screens/OnboardingGenderScreen.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react';
+import {SafeAreaView, StyleSheet, View} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import ApproachText from '../ApproachText';
+import ApproachWideButton from '../ApproachWideButton';
+import CheckBox from '@react-native-community/checkbox';
+import {OnboardingLookingForScreenName} from './OnboardingLookingForScreen';
+
+export const OnboardingGenderScreenName = 'onboardingGender';
+
+export default function OnboardingGenderScreen({navigation}) {
+  const {colors} = useTheme();
+  const styles = makeStyle(colors);
+  const tintColorsAndroid = {true: colors.primary, false: colors.primary};
+
+  const [isMale, setIsMale] = useState(false);
+  const [isFemale, setIsFemale] = useState(false);
+  const [isOther, setIsOther] = useState(false);
+
+  // Clearly NOT the way to do this, but couldn't figure anything else out at the time
+  function setOneGender(gender, value) {
+    setIsMale(false);
+    setIsFemale(false);
+    setIsOther(false);
+
+    if (gender === 'male') {
+      setIsMale(true);
+    }
+    if (gender === 'female') {
+      setIsFemale(true);
+    }
+    if (gender === 'other') {
+      setIsOther(true);
+    }
+  }
```

Comment:
You can simplify to:
```suggestion
  // Clearly NOT the way to do this, but couldn't figure anything else out at the time
  function setOneGender(gender) {
    setIsMale(gender === 'male')
    setIsFemale(gender === 'female');
    setIsOther(gender === 'other');
  }
```
Also, instead of using hard-coded strings like that, create an `Enum` for each of these values and use that instead.